packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -204,6 +204,15 @@ export class Loan {
     return await this.client.call('getvault', [vaultId], 'bignumber')
   }
 
+  /**
+   * List all available vaults.
+   *
+   * @return {Promise<Record<string, VaultDetails>>} Object including details of the vaults.
+   */
+  async listVaults (): Promise<Record<string, VaultDetails>> {
```

Comment:
Currently, there isn't a way to listVaults with pagination or filtering?

Reply:
yes. there's no filtering or pagination for this RPC atm.

Reply:
Damn, this is gonna be awkward. We need indexed address and pagination for Light Wallet.